content/community.md

```diff
@@ -28,6 +28,19 @@ In Slack, remember that channels are free. If you want to start a conversation o
 
 In general, in this community, we trust that everyone will behave in ways that they believe are appropriate. There are some hard-and-fast rules in our [code of conduct]({{< relref "code_of_conduct.md" >}}), which we expect everyone to adhere to: specifically, rules prohibiting harassment and intolerant behavior. However, in general, we prefer to have guidelines for behavior rather than hard-and-fast rules in a way that promotes organic growth and discussion. 
 
+### Solicitation
+
+This community is built for data practitioners to discuss the work that they do, the ideas that they have, and the things that they are learning. It is not intended to be lead generation for vendors or recruiters. Vendors can add enormous value to the community by being there to answer questions about their products when questions arise, but we do not want the community to become a sales-channel that drowns out legitimate discussion.
```

Comment:
would you consider replacing "legitimate discussion" with "organic discussion" or something.  I guess I'm taking issue with the idea that sales conversations are inherently illegitimate.

Reply:
Good, catch, Sam! Will make this change.